en-US/emails/2018/firefox_account_welcome_journey.lang

```diff
@@ -0,0 +1,318 @@
+Send tabs instantly
+
+
+# Body
+;Open a page on one screen and view it instantly on any of your synced devices (even across operating systems). Say goodbye to emailing yourself links.
+Open a page on one screen and view it instantly on any of your synced devices (even across operating systems). Say goodbye to emailing yourself links.
+
+
+# CTA
+;Read More
+Read More
+
+
+# Email: Pocket
+# Mockup image: https://tinyurl.com/yc23tt4s
+# Subject #1
+;Account Tips: Save your faves with Pocket.
+Account Tips: Save your faves with Pocket.
+
+
+# Subject #2
+;Save your faves with Pocket.
+Save your faves with Pocket.
+
+
+# Subject #3
+;Like bookmarks? You’ll love this Firefox Account feature.
+Like bookmarks? You’ll love this Firefox Account feature.
+
+
+# Pre-header
+;Save, read and listen on the go.
+Save, read and listen on the go.
+
+# Headline
+;Put the content you love in your Pocket.
+Put the content you love in your Pocket.
+
+
+# Body
+;Capture and enjoy the best of the web – even when the wifi is out on your commute. Your saved articles, pages and interests are always ready for you. And discover more of what you love with Pocket recommendations.
+Capture and enjoy the best of the web – even when the wifi is out on your commute. Your saved articles, pages and interests are always ready for you. And discover more of what you love with Pocket recommendations.
+
+
+# Body
+;Plus, Pocket is from Firefox, so you can sign in with your <a href="https://www.mozilla.org/firefox/accounts">Firefox Account</a>.
+Plus, Pocket is from Firefox, so you can sign in with your <a href="https://www.mozilla.org/firefox/accounts">Firefox Account</a>.
+
+
+# CTA
+;Get Pocket
+Get Pocket
+
+
+;Save to Pocket.
+Save to Pocket.
+
+
+;View when ready.
+View when ready.
+
+
+;If you use bookmarks, you’re gonna love this
+If you use bookmarks, you’re gonna love this
+
+
+# Email: Privacy
+# Mockup image: https://tinyurl.com/yc23tt4s
+# Subject #1
+;Account Tips: Check your privacy settings.
+Account Tips: Check your privacy settings.
+
+
+# Subject #2
+;How to: Stay in control with your Firefox Account
+How to: Stay in control with your Firefox Account
+
+
+# Pre-header #1
+;Stay safer online
+Stay safer online
+
+
+# Pre-header #2
+;Safety first (and second and third…)
+Safety first (and second and third…)
+
+
+# Headline
+;Stay safe and secure
+Stay safe and secure
+
+
+# Body
+;When it comes to online security, we’ve got you covered with powerful privacy settings. You’re always in control of what info is shared between which devices.
+When it comes to online security, we’ve got you covered with powerful privacy settings. You’re always in control of what info is shared between which devices.
+
+
+# Body #1
+;End-to-End Encryption
+End-to-End Encryption
+
+
+# Body Subhead #1
+;Keeps your synced data on lockdown. Even we can’t see what you share.
+Keeps your synced data on lockdown. Even we can’t see what you share.
+
+
+# Body #2
+;Tracking Protection
```

Comment:
Why are we promoting something that is not in Firefox anymore? Is marketing aware that we moved away from Tracking Protection, given this is going out in 2 months?